server/models.py

```diff
@@ -24,18 +27,66 @@ def process_result_value(self, value, engine):
         else:
             return set(value.split(','))
 
+
 class User(db.Model, UserMixin):
     __tablename__ = 'users'
     id = db.Column(db.Integer, primary_key=True)
-    email = db.Column(db.String(255), nullable=False, index=True)
-    offerings = db.Column(StringSet, nullable=False)
+    name = db.Column(db.String(255), index=True, nullable=False)
+    canvas_id = db.Column(db.String(255), nullable=False, index=True)
+    staff_offerings = db.Column(StringSet, nullable=False)
```

Comment:
we should call these course_offerings

Reply:
so, 
`staff_course_offerings` and `student_course_offerings`?